server/sonar-server/src/main/java/org/sonar/server/setting/ws/SettingsFinder.java

```diff
@@ -58,6 +58,22 @@ public SettingsFinder(DbClient dbClient, PropertyDefinitions definitions) {
       .collect(Collectors.toList());
   }
 
+  public List<Setting> loadProjectSettings(DbSession dbSession, ComponentDto project, String propertyKey) {
```

Comment:
I son't see the difference with method loadComponentSettings(). Could you explain ?

Reply:
I wanted a project specific implementation implementation, but this one while much wider, does the job i've removed mine.